test/test_device_radix_sort.cu

```diff
@@ -285,7 +285,7 @@ cudaError_t Dispatch(
 {
     return DeviceSegmentedRadixSort::SortPairs(
         d_temp_storage, temp_storage_bytes,
-        d_keys, d_values, num_items,
+        d_keys, d_values, static_cast<int>(num_items),
```

Comment:
Add a runtime check that fails the test if `num_items` doesn't fit into an `int`, since this test file also covers the `DeviceRadixSort` 64-bit offsets and we may end up passing larger `num_items` here.